modules/restoring-an-archived-pipeline-run.adoc

```diff
@@ -0,0 +1,34 @@
+:_module-type: PROCEDURE
+
+[id="restoring-an-archived-pipeline-run_{context}"]
+= Restoring an archived pipeline run
+
+[role='_abstract']
+You can restore an archived run to the active state.
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group} ) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and has a pipeline server.
+* You have imported a pipeline to an active pipeline server.
+* An archived run exists in your project. 
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
+. On the *Runs* page, select the *project* for the archived pipeline run that you want to restore. 
+. On the *Run details* page, click the *Archived* tab.
+. Click the action menu (*&#8942;*) beside the run that you want to restore and click *Restore*.
+. In the *Restore run* dialog, enter the run name in the text field to confirm that you intend to restore it.
+. Click *Restore*.
+
+.Verification
+* The restored run appears in the *Active* tab on the *Runs* page.
```

Comment:
overview of pipelines module states it is restored to its former state - so should this be *Active* or *Schedules" tab??